includes/express-checkout/class-wc-payments-express-checkout-button-handler.php

```diff
@@ -260,14 +260,39 @@ public function scripts() {
 			'is_checkout_page'   => $this->express_checkout_helper->is_checkout(),
 		];
 
-		WC_Payments::register_script_with_dependencies( 'WCPAY_EXPRESS_CHECKOUT_ECE', 'dist/express-checkout', [ 'jquery', 'stripe' ] );
+		if ( WC_Payments_Features::is_tokenized_cart_ece_enabled() ) {
```

Comment:
Creating some branching logic to enqueue the "tokenized" ECE vs the non-tokenized ECE.
The "tokenized" ECE is the copy of the files, all part of this PR.
They're not "tokenized" yet - they're just a copy of the ECE implementation.
The tokenization efforts are part of https://github.com/Automattic/woocommerce-payments/issues/9722